assets/aisforms/styles/aisforms-common/_aisformsCommon.css

```diff
@@ -181,6 +181,7 @@ div.SecureDocumentEditorRowDisplay h1 {
   padding: .125rem 1rem .1875rem;
   color: #000;
   background-color: transparent;
+  text-decoration: none;
```

Comment:
slipped this in here. I don't know when these links started being underlined, sometime recently